database/README.md

```diff
@@ -309,6 +326,56 @@ Indexes:
 - PRIMARY KEY: btree(`id`)
 - quotes\_token\_expiration: btree (`sell_token`, `buy_token`, `expiration_timestamp` DESC)
 
+### proposed\_solutions
+
+All solutions reported by solvers, that were part of a solver competition.
+
+ Column        | Type      | Nullable | Details
+---------------|-----------|----------|--------
+ auction\_id   | bigint    | not null | auction for which the solution was proposed 
+ uid           | bigint    | not null | unique id of the proposed solution within a single auction
```

Comment:
Would be useful to highlight that this uniqueness is guaranteed by generating it from a sequence in the DB.

Reply:
Currently it's not guaranteed by DB, but by the persistence layer which generates it when storing solutions. Personally I like it more this way for flexibility.